db/migrate/20171230113732_remove_default_type.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDefaultType < ActiveRecord::Migration[5.0]
+  def change
+    change_column_default(:cases, :type, 'Case::FOI')
```

Comment:
This may be a convenience for now, but I think we should probably remove this soonish.

Reply:
Actually, we don't even need this do we?